frontend/app/templates/create-badges.hbs

```diff
@@ -1,10 +1,13 @@
 <div class="ui stackable grid">
-    <div class="right floated right aligned two wide column">
-        <button {{action 'togglePreview'}} class="tiny ui basic orange button" data-tooltip="Toggle Preview" data-position="left center">
-            <i class="exchange icon"></i>
-        </button>
+    <div class="right floated right aligned two wide column" >
```

Comment:
Just run `git checkout -- frontend/app/templates/create-badges.hbs`

Reply:
That will overwrite with original file :smile: .